### PR TITLE
Add R4 instances to the RDS check

### PR DESCRIPTION
**Pull request by enokawa**
## Pull Request Checklist

**Is this in reference to an existing issue?**

#### General

- [x] Update Changelog following the conventions laid out [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)

- [ ] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [x] RuboCop passes

- [ ] Existing tests pass

#### Purpose
Support for R4 instances.

#### Known Compatibility Issues

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins
 ### Breaking Changes
 - handler-ses.rb, handler-sns.rb: Update to AWS-SDK v2. With the update to AWS-SDK v2 these handlers no longer take `access_key`, `secret_key`, or `use_ami_role` settings. Authentication should be configured per [here](https://github.com/sensu-plugins/sensu-plugins-aws/blob/master/README.md#authentication). (@eheydrick)
 
+### Added
+- check-rds.rb: Add R4 instances (@enokawa)
+
```
Comment (eheydrick) [inline CHANGELOG.md]: can you move this under the Unreleased section.

**Comment by enokawa** [comment]
@eheydrick Thank you for reviewing!! I fixed CHANGELOG.

**Comment by eheydrick** [comment]
👍 Thanks again @enokawa!

**Comment by eheydrick** [comment]
Released in 10.0.1: https://rubygems.org/gems/sensu-plugins-aws/versions/10.0.1

**Pull request merged**